pkg/osbuild/oscap_autotailor_stage.go

```diff
@@ -8,9 +8,15 @@ type OscapAutotailorStageOptions struct {
 }
 type OscapAutotailorConfig struct {
 	OscapConfig
-	NewProfile string   `json:"new_profile"`
-	Selected   []string `json:"selected,omitempty"`
-	Unselected []string `json:"unselected,omitempty"`
+	NewProfile string                    `json:"new_profile"`
+	Selected   []string                  `json:"selected,omitempty"`
+	Unselected []string                  `json:"unselected,omitempty"`
+	Overrides  []OscapAutotailorOverride `json:"overrides,omitempty"`
+}
+
+type OscapAutotailorOverride struct {
+	Var   string      `json:"var"`
+	Value interface{} `json:"value"`
```

Comment:
I think that it would be good to ensure that the `Value` can be only `int` or `string`. This way, one could set anything as the value (`bool`, `nil`, etc..).

Probably an equivalent of https://github.com/osbuild/images/blob/82de94ad6cf7979156e274e1d5e9c94d718e6f62/pkg/osbuild/pam_limits_conf_stage.go#L79-L102

Reply:
I did a simple implementation, but I think the above is more elegant. I'll come back to this.

Reply:
The validation check is better than nothing :slightly_smiling_face: Thanks!